upup/pkg/fi/cloudup/awstasks/launchtemplate_target_terraform.go

```diff
@@ -262,55 +265,42 @@ func (t *LaunchTemplate) RenderTerraform(target *terraform.TerraformTarget, a, e
 			}
 		}
 	}
+
 	devices, err := e.buildRootDevice(cloud)
 	if err != nil {
 		return err
 	}
-	for n, x := range devices {
-		tf.BlockDeviceMappings = append(tf.BlockDeviceMappings, &terraformLaunchTemplateBlockDevice{
-			DeviceName: fi.PtrTo(n),
-			EBS: []*terraformLaunchTemplateBlockDeviceEBS{
-				{
-					DeleteOnTermination: fi.PtrTo(true),
-					Encrypted:           x.EbsEncrypted,
-					KmsKeyID:            x.EbsKmsKey,
-					IOPS:                x.EbsVolumeIops,
-					Throughput:          x.EbsVolumeThroughput,
-					VolumeSize:          x.EbsVolumeSize,
-					VolumeType:          fi.PtrTo(string(x.EbsVolumeType)),
-				},
-			},
-		})
+
+	devicesKeys := getSortedKeysFromMap(devices)
+	for _, key := range devicesKeys {
+		v := devices[key]
+		terraformLaunchTemplateBlockDevice := createTerraformLaunchTemplateBlockDevice(key, v)
+		tf.BlockDeviceMappings = append(tf.BlockDeviceMappings, terraformLaunchTemplateBlockDevice)
```

Comment:
This should not be needed starting Go 1.22:
https://go.dev/blog/loopvar-preview
```suggestion
		tf.BlockDeviceMappings = append(tf.BlockDeviceMappings, createTerraformLaunchTemplateBlockDevice(key, devices[key]))
```

Reply:
@hakman Yes, the temp vars are not needed, but I think the code is easier to read that way because 
```
tf.BlockDeviceMappings = append(tf.BlockDeviceMappings, createTerraformLaunchTemplateBlockDevice(key, devices[key]))
```
is a line too long IMHO
WDYT ?

Reply:
Sure, but at least inline the `v := devices[key]` part.

Reply:
That sounds good 🙂 , doing so!

Reply:
Done!